client/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "client",
   "engines": {
-    "node": "8.16.0"
+    "node": "12.16.2"
   },
   "version": "0.1.0",
   "private": true,
   "proxy": "http://localhost:3001",
   "dependencies": {
     "axios": "^0.19.2",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "^3.4.0"
+    "react-scripts": "^3.4.1"
```

Comment:
Were these done manually or automatically? There shouldn't be a difference between what resolves for these changes in the package-lock.json.

Reply:
I honestly don't remember updating them, but I reverted them just in case. I ran `npm update` and the lock file didn't change so I think we're good to leave them at the `.0` versions.